participants/yituyun.md

```diff
@@ -0,0 +1,67 @@
+# DBC-AI-Distributed-Cloud-Storage
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name:
+    - *yituyun 忆图云*
+
+- Team Slack Handles:
+    - *litai686@qq.com*
+
+## (1) Brief Project Description
+
+*An open image search library, where everyone is a picture provider and contributor, is stored on the Filecoin Network.*
+
+## (2) Link to Project UI
+
+*https://github.com/litai686/imgSearch.git*
+*http://f.etu1314.com*
+
+## (3) What does your application/UI do?
+
+*An open image search library, where everyone is a picture provider and contributor, is stored on the Filecoin Network.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*We didn't use the data.*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*This project is an open photo gallery. If we find any copyright infringement, we will delete it in time. We are only responsible for providing technical support and platform management*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*We hope to upload about 50TB of data. At present*
+
+
+## (7) How are you structuring the data?
+
+*We will upload the user's original data directly, and the specific data format is determined by the user himself.*
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*Users upload data directly through the web page, and then we call the API interface of filecoin to transfer the data to the filecoin network. Our own server will record the user's data CID, which is convenient for users to search and download data.*
+
+## (9)  What tech stack will you use to store the data?
+
+*Lotus\ipfs*
```

Comment:
Powergate has a built-in Lotus and IPFS node if this is useful for your purposes.